front/app/containers/AggDropDown/AggDropDown.tsx

```diff
@@ -568,41 +435,26 @@ class AggDropDown extends React.Component<AggDropDownProps, AggDropDownState> {
     const title = aggToField(agg);
     const icon = `chevron${isOpen ? '-up' : '-down'}`;
     return (
-      <SiteProvider>
-        {site => (
-          <ApolloConsumer>
-            {apolloClient => (
-              <PanelWrapper>
-                <Panel
-                  onToggle={this.handleToggle}
-                  expanded={isOpen}
-                  className="bm-panel-default">
-                  <Panel.Heading className="bm-panel-heading">
-                    <Panel.Title className="bm-panel-title" toggle>
-                      <div className="flex">
-                        <span>{title}</span>
-                        <span>
-                          <FontAwesome name={icon} />{' '}
-                        </span>
-                      </div>
-                    </Panel.Title>
-                  </Panel.Heading>
-                  {isOpen && (
-                    <Panel.Collapse className="bm-panel-collapse">
-                      <Panel.Body>{this.renderFilter()}</Panel.Body>
-                      <Panel.Body>
-                        {this.renderBucketsPanel(apolloClient, site.siteView)}
-                      </Panel.Body>
-                    </Panel.Collapse>
-                  )}
-                </Panel>
-              </PanelWrapper>
-            )}
-          </ApolloConsumer>
-        )}
-      </SiteProvider>
+      <PanelWrapper>
```

Comment:
note the improved legibility of the component by using higher-order components to pass site and client into the props of this component.